### DISCUSSION TITLE
Error: Data P must be a multiple of 16 long

### DISCUSSION THREAD

**Discussion by yangwcool** [remotely-save/remotely-save#823] [Q&A]
云端存储为七牛云S3，开启端到端加密。前一天在家里的笔记本创建vault并上传，当天在手机端同步成功。第二天在办公室的笔记本同步，提示`error while sync`。打开控制台看了一下，提示：`Error: Data P must be a multiple of 16 long, is 27。`    

![screen-capture](https://github.com/user-attachments/assets/ebc77ffd-9e01-4dbf-9acf-1086e3e2979d)

**Comment by fyears** [comment]
密码和加密方式确认都对齐了吗？

**Reply by yangwcool** [reply]
感谢大佬回复。  
我着急用就换成腾讯云了，目前一切正常。  
密码没问题，我核对过好几遍。加密方式没修改过。密码里有一个下划线，不知道是不是这个原因。

**Comment by fyears** [comment]
换成腾讯云正常，那么就不是插件问题了，加密算法是相同的代码哇。可能是七牛云的鉴权判定有点问题。不过我没有七牛云，无法复现。先用着腾讯云吧。

**Comment by zeng-y-l** [comment]
我用OneDrive，也遇到相同的问题，我把那个rs-test开头的文件夹删了就行了
